src/kOS/Function/Misc.cs

```diff
@@ -373,4 +373,35 @@ public override void Execute(SharedObjects shared)
             ReturnValue = shared.Cpu.DumpVariables();
         }
     }
+
+    [Function("warpto")]
+    public class WarpTo : FunctionBase
+    {
+        public override void Execute(SharedObjects shared)
+        {
+            int args = CountRemainingArgs(shared);
+            double maxWarp = 8.0;
+            double minWarp = 2.5;
+            double ut = 0.0;
+            switch (args)
+            {
+                case 3:
+                    minWarp = GetDouble(PopValueAssert(shared));
+                    maxWarp = GetDouble(PopValueAssert(shared));
+                    ut = GetDouble(PopValueAssert(shared));
+                    break;
+                case 2:
+                    maxWarp = GetDouble(PopValueAssert(shared));
+                    ut = GetDouble(PopValueAssert(shared));
+                    break;
+                case 1:
+                    ut = GetDouble(PopValueAssert(shared));
+                    break;
+                default:
+                    break;
```

Comment:
I forgot to move the case 1 logic into the default switch.  That's why I made sure I didn't set the timewarp until after we assert the bottom args.  In which case, passing zero args will throw an exception when trying to pop, and passing too many args will throw an exception with the assert before triggering the timewarp.  Unless you would prefer an explicit throw in the switch.